x-pack/plugins/cases/public/components/create/case_owner_selection.tsx

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo, useCallback, useEffect } from 'react';
+
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFormRow,
+  EuiIcon,
+  EuiKeyPadMenu,
+  EuiKeyPadMenuItem,
+  useGeneratedHtmlId,
+} from '@elastic/eui';
+
+import { euiStyled } from '../../../../../../src/plugins/kibana_react/common';
+import { FieldHook, getFieldValidityAndErrorMessage, UseField } from '../../common/shared_imports';
+
+interface MenuSelectionProps {
+  field: FieldHook<string>;
+  isLoading: boolean;
+  availableOwners: string[];
+}
+
+interface Props {
+  availableOwners: string[];
+  isLoading: boolean;
+}
+
+const SECURITY_SOLUTION = 'securitySolution';
+const OBSERVABILITY = 'observability';
```

Comment:
We already have `SECURITY_SOLUTION_OWNER` constant in https://github.com/elastic/kibana/blob/e823d96963fa94797fdaee488d118b1a262a4154/x-pack/plugins/cases/common/constants.ts#L89
We could add `OBSERVABILITY_OWNER` and load both from there.

We could create a new object `ownerInfo` in the constants.ts as well (or any other name you like), to define the `label` and the `iconType` of each owner, so we have everything in one single place. 

Then, having that, it would also be nice to create the `EuiKeyPadMenuItem`s iteratively, we could define a `const DEFAULT_SELECTABLE_OWNERS = [SECURITY_SOLUTION_OWNER, OBSERVABILITY_OWNER] as const;` here in the component itself. This way it will be easy to allow a `selectable_owners` prop in the future if we ever need to display other owners here.

Reply:
I agree with @semd. The information should be in one place.

Reply:
fixed!